_includes/utilities/values-intro.html

```diff
@@ -18,4 +18,4 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
 </div>
 
 <h3 class="font-lang-4 margin-bottom-0">Palettes</h3>
-<p class="utility-text margin-top-1">Palettes are predefined groups of values. Add palettes to the <code>$[utility_family]-palettes</code> list variable in  <code>_uswds-theme-utilities.scss</code> to add those values to the utility family.</p>
+<p class="utility-text margin-top-1">Palettes are predefined groups of values. Add palettes to the <code>$[utility_family]-palettes</code> list variable in  <code>_uswds-theme.scss</code> to add those values to the utility family.</p>
```

Comment:
Looks like there are two spaces here.
```suggestion
<p class="utility-text margin-top-1">Palettes are predefined groups of values. Add palettes to the <code>$[utility_family]-palettes</code> list variable in <code>_uswds-theme.scss</code> to add those values to the utility family.</p>
```

Reply:
I noticed this in a few spots actually but thought maybe it was intentional before the code blocks. I'll remove!